hmpps_person_match/db/migrations/versions/dadb198e98f6_add_manual_override_and_override_scopes_.py

```diff
@@ -0,0 +1,28 @@
+"""Add manual_override and override_scopes to person table
```

Comment:
I generated this in the dev environment with:

```
DB_SSL_ENABLED=False \                                           
DATABASE_USERNAME=root \
DATABASE_PASSWORD=dev \
DATABASE_HOST=localhost \
DATABASE_PORT=5432 \
DATABASE_NAME=postgres \
uv run alembic revision -m "Add manual_override and override_scopes to person table"
```

I've never used `alembic` before so I don't know if this is correct